internal/tracker/bundle_info.go

```diff
@@ -71,31 +72,56 @@ func newBundleInfo(ctx context.Context, ref image.ImageReference, tasks commonTa
 	return &info, nil
 }
 
-// updateCommonTasks updates the commonPipelineTasks attributes with the tasks found
-// in the given pipeline.
-func (info *bundleInfo) updateCommonTasks(ctx context.Context, pipelineName string) error {
+// getBundlePipeline gets a pipeline from a bundle by the pipeline name
+func (info *bundleInfo) getBundlePipeline(ctx context.Context, pipelineName string) (error, v1beta1.PipelineObject) {
```

Comment:
Odd to see it the other way around

```suggestion
func (info *bundleInfo) getBundlePipeline(ctx context.Context, pipelineName string) (v1beta1.PipelineObject, error) {
```

Reply:
Yes, good catch. fixed